eng/pipelines/common/xplat-setup.yml

```diff
@@ -170,12 +170,12 @@ jobs:
 
         # OSX Public Build Pool (we don't have on-prem OSX BuildPool).
         ${{ if and(in(parameters.osGroup, 'osx', 'maccatalyst', 'ios', 'iossimulator', 'tvos', 'tvossimulator'), eq(variables['System.TeamProject'], 'public')) }}:
-          vmImage: 'macos-13'
+          vmImage: 'macos-latest'
```

Comment:
We don't want these to change underneath us. Using the latest available (15) is good, but we shouldn't use the "latest" tag